pennylane_lightning/lightning_gpu/_adjoint_jacobian.py

```diff
@@ -61,8 +67,23 @@ class LightningGPUAdjointJacobian(LightningBaseAdjointJacobian):
 
     # pylint: disable=too-few-public-methods
 
-    def __init__(self, qubit_state: LightningGPUStateVector, batch_obs: bool = False) -> None:
+    def __init__(
+        self,
+        qubit_state: LightningGPUStateVector,
+        batch_obs: bool = False,
+        use_mpi: bool = False,
+        mpi_handler=None,
```

Comment:
How come the py formatter didn't capture this? :thinking: 
```suggestion
        batch_obs: bool = False,
        use_mpi: bool = False,
        mpi_handler = None,
```